static/js/zamboni/analytics.js

```diff
@@ -1,14 +1,31 @@
 // GA Analytics code. The '_setAccount' below is specific to AMO tracking.
 
+function isDoNotTrackEnabled() {
+    // We ignore things like `msDoNotTrack` because they are for older,
+    // unsupported browsers and don't really respect the DNT spec. This
+    // covers new versions of IE/Edge, Firefox from 32+, Chrome, Safari, and
+    // any browsers built on these stacks (Chromium, Tor Browser, etc.).
+    var dnt = navigator.doNotTrack || window.doNotTrack;
+    if (dnt === '1') {
+        window.console && console.info('[TRACKING]: Do Not Track Enabled; Google Analytics will not be loaded.');
+        return true;
+    }
+
+    // Known DNT values not set, so we will assume it's off.
+    return false;
+}
+
 var _gaq = _gaq || [];
```

Comment:
Ah I see - this is outside the if block because various events push onto this array but it's just a no-op without the script inserted.